src/main/java/org/dcsa/bkg/service/impl/BookingServiceImpl.java

```diff
@@ -151,6 +151,8 @@
     Booking requestedBooking = bookingMapper.dtoToBooking(bookingRequest);
     requestedBooking.setDocumentStatus(DocumentStatus.RECE);
     requestedBooking.setBookingRequestDateTime(OffsetDateTime.now());
+    requestedBooking.setCreatedDateTime(OffsetDateTime.now());
```

Comment:
I have just noticed now - that CreatedDateTime and bookingRequestDateTime is one and the same thing... :-(
We should delete CreatedDateTime - as it is not needed - sorry :-(
Please make a variable for the offsetDateTime - so both created and updated is the same (alternatively set updatedDateTime = bookingRequestDateTime ;-))